mllib/src/main/scala/org/apache/spark/mllib/tree/loss/LogLoss.scala

```diff
@@ -52,4 +51,10 @@ object LogLoss extends Loss {
     // The following is equivalent to 2.0 * log(1 + exp(-margin)) but more numerically stable.
     2.0 * MLUtils.log1pExp(-margin)
   }
+
+  override private[spark] def computeProbability(prediction: Double): Double = {
+    // The probability can be calculated as:
```

Comment:
This doc is just repeating the code line below it.  Instead, how about documenting in Scala doc for this method that it returns the estimated probability of a label of 1.0?

also, rename prediction -> rawPrediction